packages/cxl-lumo-styles/scss/global.scss

```diff
@@ -174,6 +174,14 @@ ul {
   @include mixins.wrap();
 }
 
+vaadin-button[class~="block-xs"] {
+  display: block;
+
+  @media #{mq.$medium} {
+    display: inline-block;
+  }
+}
+
```

Comment:
Is it your invention or taken from upstream? Isn't it possible to style buttons accordingly within the component instead of introducing global rule?

Reply:
My invention. Maybe @freudFlintstone knows a better solution for how to turn buttons inside a slotted element into blocks on mobile?

Reply:
I imagine this is related to the buttons under the certificate? If so:

I always prefer controlling layout from the container. The buttons should be inside a `display: flex` container styled from the component's scope CSS file, no need to add global CSS targeting `vaadin-button`. Remember we should avoid global scope if possible. 
Using a flex container, the buttons will naturally wrap if the viewport is too narrow. If you want them to always stack in mobile, use `flex-direction: column` by default, then change it to` row` on the desired breakpoint (prefer 528px - small):
```
.container {
    @media #{mq.$small} {
         flex-direction: row;
     }
}
```